hip/static/styles/includes/header.scss

```diff
@@ -45,6 +45,8 @@
 .header-btn-hip {
   @include light-blue-bg;
   border-radius: 0;
+  text-transform: uppercase;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
Should we also select the `button` element to apply line 48 and line 49 across all buttons. Or is this restricted to just the header buttons?

Reply:
Good point. I'll make sure it applies to all buttons